docs/scanning-vendors.md

```diff
@@ -0,0 +1,457 @@
+
+## Package Identification and purls
+In order to accurately report on vulnerabilities in Red Hat products, scanning vendors must properly identify the 
+Red Hat package versions for RPMs, RPM modules and container first content, to identify any backported fixes. The
+following section provides guidance on how to find information about packages and how they are reported in Red Hat's 
+CSAF advisories and VEX  content. 
+
+### RPMs and RPM modules 
+An RPM package is a file format used by the Red Hat Package Manager (RPM) system for software distribution and management, 
+which package consists of an archive of files and metadata used to install and erase these files.
+
+There are two types of RPM packages: source RPMs and binary RPMs. Both types share the same file format and tooling, but have
+different contents and serve different purposes. A source RPM (SRPM) contains source code and a spec file, which 
+describes how to build the source code into a binary RPM, while a binary RPM contains the binaries built from the 
+sources and patches.
+
+Similarly, an RPM module is a set of RPM packages that represent a component and are usually installed together. A typical module 
+contains packages with an application, packages with the application-specific dependency libraries, packages with 
+documentation for the application, and packages with helper utilities. 
+
+Both binary RPMs and RPM modules installed in a container image can be discovered using the `rpm -qa` command.
+```
+# Example return of RPM query
+$ rpm -qa
+
+libgcc-11.3.1-4.3.el9.x86_64
+subscription-manager-rhsm-certificates-20220623-1.el9.noarch
+setup-2.13.7-9.el9.noarch
+filesystem-3.16-2.el9.x86_64
+basesystem-11-13.el9.noarch
+```
+
+Additionally, SRPMs can be discovered from a binary RPM by using the following command: 
+ ```
+# Example return of SRPM query
+$ rpm -q --qf "%{SOURCERPM}\n" libgcc-11.3.1-4.3.el9.x86_64    
+   
+gcc-11.3.1-4.3.el9.src.rpm
+ ```
+SRPMS, RPMs and RPM modules are represented in CSAF advisories and VEX data using the `rpm` purl type. 
+
+```
+# Example of a RPM purl
+pkg:rpm/redhat/libgcc-11.3.1-4.3.el9?arch=x86_64
+
+# Example of a SRPM purl 
+pkg:rpm/redhat/gcc?arch=src
+```
+
+More detailed information about RPM purl usage can be found [here](https://redhatproductsecurity.github.io/security-data-guidelines/purl/#identifying-rpm-packages).
+
+### Container metadata and container first content 
+Container images frequently include non-RPM packages, often referred to as container first content. Non-RPM packages 
+that are present found in a container image will be reported on the container itself instead of the package name. Containers 
+will use the `oci` purl type. 
+
+```
+# Example of a container purl 
+pkg:oci/ose-console-rhel9?repository_url=registry.redhat.io/openshift4/ose-console-rhel9
```

Comment:
It is important to mention that such container purl representation is when the associated status is different than FIXED. When something is fixed, the purl for such container image includes digest sha and image tag, like:  
`pkg:oci/ose-cloud-event-proxy-rhel9@sha256:bc59c5fb6e9f44b155c40f4e1ec25a7d884b550d1007252ea75341dee79d7c9f?arch=ppc64le&repository_url=registry.redhat.io/openshift4/ose-cloud-event-proxy-rhel9&tag=v4.17.0-202410151605.p0.g0e1a8c3.assembly.stream.el9"`